sdk/eventhub/Azure.Messaging.EventHubs/src/Processor/PartitionManager.cs

```diff
@@ -42,6 +42,21 @@ public abstract class PartitionManager
         ///
         public abstract Task<IEnumerable<PartitionOwnership>> ClaimOwnershipAsync(IEnumerable<PartitionOwnership> partitionOwnership);
 
+
+        /// <summary>
```

Comment:
This class should be going away now that there's no plugin model, no?

Reply:
Separate PR for moving, removing and flipping class to `internal` :)